x-pack/legacy/plugins/infra/public/pages/metrics/components/node_details_page.tsx

```diff
@@ -41,18 +41,14 @@ interface Props {
   isAutoReloading: boolean;
   refreshInterval: number;
   sideNav: NavItem[];
-  metadata: InfraMetadata | null;
+  metadata: InfraMetadata;
   addNavItem(item: NavItem): void;
   setRefreshInterval(refreshInterval: number): void;
   setAutoReload(isAutoReloading: boolean): void;
   triggerRefresh(): void;
   setTimeRange(timeRange: MetricsTimeInput): void;
 }
 export const NodeDetailsPage = (props: Props) => {
-  if (!props.metadata) {
```

Comment:
:information_source: This condition broke the hook call order invariant so it was moved to the parent component.